ui/js/controllers/main.js

```diff
@@ -62,7 +62,7 @@ treeherderApp.controller('MainCtrl', [
             'u',     // Display only unclassified failures
             'b',     // Pin selected job and add related bug
             'c',     // Pin selected job and add classification
-            'f',     // Enter a custom job or platform filter
+            'f',     // Enter a custom job or quick filter
```

Comment:
We can probably say here `// Enter a quick filter` the same as we do in our Help.